llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1164,10 +1209,15 @@ void MCCASBuilder::startSection(const MCSection *Sec) {
   const MCSection *DebugInfoSection =
       Asm.getContext().getObjectFileInfo()->getDwarfInfoSection();
 
+  auto DwarfLineSection =
+      Asm.getContext().getObjectFileInfo()->getDwarfLineSection();
+
   if (RelocLocation == Atom) {
     // Build a map for lookup.
     for (auto R : ObjectWriter.getRelocations()[Sec]) {
-      if (R.F && Sec != DebugInfoSection)
+      // For the Dwarf Line Section, just append the relocations to the
```

Comment:
I think the comment is not reflecting the line below, as it only mentions the Line sections, but the line below refers to both debug info and debug section.

I think we could either remove the comment or make it more generic and just say "Dwarf Section"

Reply:
Yep! Good catch!